azure-quantum/azure/quantum/target/target.py

```diff
@@ -132,7 +132,22 @@ def submit(
         :return: Azure Quantum job
         :rtype: Job
         """
+
         input_params = input_params or {}
+
+        # If the input_data is `QSharpCallable` (coming from the IQ# `qsharp` Python Package)
+        # we need to convert it to QIR bitcode and set the necessary parameters for a QIR job.
+        if (input_data and type(input_data).__name__ == "QSharpCallable"):
```

Comment:
The description says that we will attempt a import qsharp if type is QsharpCallable, but I don't see that happening here. Is that intentional?

Reply:
I updated the description. The import is not needed.